code/game/objects/structures/signs/signs_departments.dm

```diff
@@ -157,6 +162,16 @@
 	desc = "A sign labelling the security department of the station. Guilty until proven innocent."
 	icon_state = "minskysec"
 
+/obj/structure/sign/departments/minsky/security/command
+	name = "Command Department"
+	desc = "A sign labelling the Command department of the station. Ask the helpful HOP for AA"
+	icon_state = "minskyhead"
+
+/obj/structure/sign/departments/minsky/security/evac
+	name = "Evacuation lounge"
+	desc = "A sign labelling the evacuation lounge of the station. A watch ETA never arrives"
```

Comment:
these two descs need a period at the end. Also change "watch" to "watched", capitalize "Lounge" in the name